addons/binding/org.openhab.binding.systeminfo.test/src/test/groovy/org/openhab/binding/systeminfo/test/SysteminfoOSGiTest.groovy

```diff
@@ -230,16 +254,32 @@ class SysteminfoOSGiTest extends OSGiTest{
     }
 
     @Test
+    public void 'assert thing status is uninitialized when there is no systeminfo service provided' () {
+        //Unblind the mock service to verify the systeminfo thing will not be initialized when no systeminfo service is provided
+        systeminfoHandlerFactory.unbindSystemInfo(mockedSystemInfo)
+
+        ThingTypeUID thingTypeUID = SysteminfoBindingConstants.THING_TYPE_COMPUTER;
+        ThingUID thingUID = new ThingUID(thingTypeUID,DEFAULT_TEST_THING_NAME);
+
+        systemInfoThing = ThingBuilder.create(thingTypeUID,thingUID).build();
+        managedThingProvider.add(systemInfoThing)
+
+        waitForAssert({
+            assertThat "Thing is not initilized if systeminfo service is missing", systemInfoThing.getStatus(),
```

Comment:
Typo initialised, I also do not completely understand the relation between the text of the assertion and the actual assertion